contracts/token/FortaCommon.sol

```diff
@@ -17,8 +17,6 @@ abstract contract FortaCommon is AccessControlUpgradeable, ERC20VotesUpgradeable
     
     bool public whitelistDisabled; // __gap[50] -> __gap[49]
```

Comment:
Can we add a comment saying this is deprecated now?

Reply:
What about changing the name to something like `deprecated_whitelistDisabled` ?

Reply:
Works for me. We will need to add the following to appease the upgrade validation:

```
/// @custom:oz-renamed-from whitelistDisabled
```